scripts/ait_bootstrap.sh

```diff
@@ -70,34 +106,27 @@ EOM
 mkvirtualenv ait
 workon ait
 
-# Pull assets, config, and secrets from s3/sm
+# Pull assets, config, and secrets from s3/sm (NEEDS TO MOVE)
 mkdir -p $SETUP_DIR
-/usr/local/aws-cli/v2/current/bin/aws --region $AWS_REGION s3 sync s3://$CONFIG_BUCKET_NAME/ait/ $SETUP_DIR/
-/usr/local/aws-cli/v2/current/bin/aws --region $AWS_REGION s3 cp s3://"$CONFIG_BUCKET_NAME"/modules/openmct-static.tgz - | tar -xz -C /var/www/html
-
+/usr/local/aws-cli/v2/current/bin/aws --region $AWS_REGION s3 sync s3://$CONFIG_BUCKET_NAME/configs/ait/ $SETUP_DIR/
+/usr/local/aws-cli/v2/current/bin/aws --region $AWS_REGION s3 cp s3://"$CONFIG_BUCKET_NAME"/configs/modules/openmct-static.tgz - | tar -xz -C /var/www/html
 
 # Install open-source AIT components
-git clone https://github.com/NASA-AMMOS/AIT-Core.git $PROJECT_HOME/AIT-Core
 cd $PROJECT_HOME/AIT-Core/
 git checkout 2.3.5
```

Comment:
idk if all these git checkout's will work if you are just downloading the static release tarballs from github (as you do in efs-bootstrap lambda); have you confirmed functionality?